modules/org.restlet.ext.apispark/src/org/restlet/ext/apispark/internal/conversion/swagger/v2_0/Swagger2Translator.java

```diff
@@ -581,4 +617,523 @@ public static Swagger getSwagger(Definition definition) {
+        }
+
+        for (String key : swaggerOperation.getResponses().keySet()) {
+            Response swaggerResponse = swaggerOperation.getResponses().get(key);
+            org.restlet.ext.apispark.internal.model.Response response =
+                    new org.restlet.ext.apispark.internal.model.Response();
+
+            Integer statusCode = null;
+            try {
+                statusCode = Integer.parseInt(key);
+                response.setCode(statusCode);
+            } catch (Exception e) {
+                // TODO: what to do with "Default" responses ?
+                LOGGER.warning("Response " + key + " for operation " + swaggerOperation.getOperationId() +
+                        " could not be retrieved because its key is not a valid status code.");
+                continue;
+            }
+
+            response.setMessage(swaggerResponse.getDescription());
+            response.setName(statusCode.toString().matches("2[0-9]{2}") ? "Success" : "Error " + statusCode);
+
+            fillOutputPayload(swaggerResponse, response, swaggerOperation, contract);
+
+            operation.getResponses().add(response);
+        }
+    }
+
+    private static void fillOutputPayload(Response swaggerResponse,
+            org.restlet.ext.apispark.internal.model.Response response,
+            Operation swaggerOperation, Contract contract) {
+        Property property = swaggerResponse.getSchema();
+        if (property == null) {
+            return;
+        }
+
+        PayLoad payload = new PayLoad();
+
+        payload.setDescription(property.getDescription());
+        payload.setArray(property instanceof ArrayProperty);
+        payload.setType(SwaggerTypes.toDefinitionType(property));
+
+        Representation representation = contract.getRepresentation(payload.getType());
+        if (representation != null) {
+            representation.addSections(swaggerOperation.getTags());
+        }
+
+        response.setOutputPayLoad(payload);
+    }
+
+    private static void fillInputPayload(Operation swaggerOperation,
+            org.restlet.ext.apispark.internal.model.Operation operation,
+            Contract contract) {
+        BodyParameter bodyParameter = SwaggerUtils.getInputPayload(swaggerOperation);
+
+        if (bodyParameter != null) {
+            PayLoad payload = new PayLoad();
+            Model model = bodyParameter.getSchema();
+            payload.setDescription(model.getDescription());
+
+            if (model instanceof ArrayModel) {
+                ArrayModel arrayModel = (ArrayModel) model;
+                payload.setArray(true);
+                payload.setType(SwaggerTypes.toDefinitionType(arrayModel.getItems()));
+
+            } else if (model instanceof RefModel) {
+                RefModel refModel = (RefModel) model;
+                payload.setType(refModel.getSimpleRef());
+
+            } else {
+                LOGGER.warning("Unsupported input payload type: " + model.getClass().getSimpleName());
```

Comment:
Good question. 
The only remaining type is ComposedModel which I don't know how to deal with.